Bio/motifs/__init__.py

```diff
@@ -20,9 +20,19 @@
 from urllib.parse import urlencode
 from urllib.request import urlopen, Request
 
+from Bio import BiopythonWarning
+from Bio.Seq import Seq
+
 
 def create(instances, alphabet="ACGT"):
     """Create a Motif object."""
+    if isinstance(instances, Seq) or isinstance(instances, str):
+        warnings.warn(
+            "instances should be iterator of Seq objects or strings."
+            "If a single sequence is given, will treat each character as a "
+            "separate sequence.",
+            BiopythonWarning,
+        )
```

Comment:
Shouldn't this be in `Instances.__init__`?

Reply:
Also, `Seq` and `str` can be checked both with a single call to `isinstance`. We should also include `MutableSeq`.

Reply:
You are right in both, I also realized I misread the issue and it's supposed to raise an error and not a warning.